libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -26,13 +26,21 @@ export class DelayedJob extends Model<DelayedJob> {
 
   @AllowNull
   @Column(INTEGER)
-  total_content: number | null;
+  totalContent: number | null;
 
   @AllowNull
   @Column(INTEGER)
-  processed_content: number | null;
+  processedContent: number | null;
 
   @AllowNull
   @Column(STRING)
-  proccess_message: string | null
+  progressMessage: string | null
+
+  @AllowNull
+  @Column(STRING)
+  createdBy: string | null;
+
+  @Column(BOOLEAN)
+  isCleared: boolean;
```

Comment:
I'm curious about this new column: when will it get set to `false`?

Reply:
There is a button in the frontend inside the notification modal. We need to display all jobs, including completed ones, but once cleared, completed jobs should no longer be shown.

Reply:
And where does the button call to? All APIs directly related to delayed jobs should be here in the jobs service.

Reply:
Adding the endpoint for the cleared ones.
You are right, this is only intended to display certain ones that comes from the site. 
For this case, what would you advice? Adding a type for delayed job? or name ?

Reply:
The endpoint to clear is done

Reply:
> Adding the endpoint for the cleared ones. You are right, this is only intended to display certain ones that comes from the site. For this case, what would you advice? Adding a type for delayed job? or name ?

Maybe making sure `isClear` defaults to `true` is enough, and then the jobs that _should_ show up here set it to `false` when they're created. It's a bit weird though. Maybe changing the name to `isAcknowledged` would make it a bit more clear. This is a new pattern to me. When I've used systems like this in the past, there was never a need to have a panel that shows the progress of old jobs. For a system like this, the majority of jobs should finish within 30 seconds or a minute, and having a panel to show progress of them is unusual.

Reply:
Great, then in API will change that so the default and change the attrib name.